### PR TITLE
first draft of fictional project READMEs

### PR DESCRIPTION
**Pull request by emckean**

### CODE REVIEW

File: README2.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Chronologue is an open source calendrical toolkit for time travel, including various utilities for developers working on time travel technology, including but not limited to time-travel devices (“time machines”), farviewers, and other IoT (internet of time) or “time-tech” applications, to convert between calendrical systems.
+
+Although the major calendrical systems (e.g. the Gregorian calendar, the Hijra calendar, Unix time) are adequate for most temporally-sequential use cases, they are not well-adapted for activities that involve traversing timestreams. Chronologue tools and libraries help developers convert between, among, and across traditional calendrical systems.
+
+Chronologue includes:
+
+* The Chronologue Time Standard (CTS), which provides consistent time coordinates across timelines
+* Test suites for integrations or applications using the Chronologue Time Standard
+* Tools for conversion between the Gregorian, Mayan, and Hijri calendars and the Chronologue Time Standard (CTS) 
+
```
Comment (ddbeck) [inline README2.md:14]: It'd be nice to know, probably right here, whether Chronologue supports self-crossing timestreams (for example—warning _The Good Place_ spoilers—the [Jeremy Bearimy](https://www.youtube.com/watch?v=RFm9ClqlGuo&feature=youtu.be&t=32) timeline). For simplicity, I'd prefer to declare such timestreams out of scope.

File: README2.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+## Using Chronologue
+
+Please see our [LICENSE](/LICENSE) file (CC-BY  4.0).
+
+Please note: closed-source conversion tools between the Chronologue Time Standard and proprietary calendars (for example, Swatch Internet Time [also known as ‘[.beat time](https://en.wikipedia.org/wiki/Swatch_Internet_Time)’]) are explicitly permitted.
```
Comment (ddbeck) [inline README2.md:19]: I found this kinda hard to follow. How about something like this?

```suggestion
Please note: you are explicitly permitted to create closed-source conversion tools between the Chronologue Time Standard and proprietary calendars (for example, Swatch Internet Time, also known as ‘[.beat time](https://en.wikipedia.org/wiki/Swatch_Internet_Time)’).
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Chronologue Project
```
[the remark this answers is not in the document]
Comment (emckean) [reply README.md:1]: Oh, good catch. I prefer Chronologue as a standalone, too. Will fix.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+----------------
+by [The Good Docs Project](thegooddocsproject.dev)
+
+The Chronologue project is a sample, completely fictional open source project used to create examples of typical open source project documentation.
+
+Chronologue helps you see what the The Good Docs Project [templates](https://www.github.com/thegooddocsproject/templates) look like in action, with actual content.
+
+### Why a Fictional Project?
```
[the remark this answers is not in the document]
Comment (emckean) [reply README.md:9]: yeah, I think you're right -- sentence case is better, and also obviates the need to go check a style manual to figure out which words DON'T get capped 😂

File: README.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+Please, use Chronologue for entertainment purposes only! 
+
+Actually, you are free to link to, reproduce, or repurpose this text, within the bounds of our [license](/LICENSE).
+
+We hope you find Chronologue useful as a way to explore technical writing and open source project documentation using templates.
+
+### How Can I Contribute to Chronologue?
+
+Please see the [Contributors Guide](/CONTRIBUTING.md) (TK).
```
[the remark this answers is not in the document]
Comment (emckean) [reply README.md:25]: 👍

**Review by flicstar — APPROVED** [review]

**Comment by flicstar** [comment]
Forgot to add a comment on this. Merging this PR to get the ball rolling on the Chronologue work.
